src/main.rs

```diff
@@ -308,6 +308,19 @@ fn main() -> BError {
                 Effectors::DoubleDamage,
             ],
         ),
+        SkillDefinition::new(
+            Skills::DoubleAttackSpeed,
+            String::from("Double Attack Speed"),
+            String::from("double_attack_speed"),
+            String::from("Double the attack speed."),
+            0.0,
+            true,
+            vec![],
+            vec![(Items::Coffee, 1,UseMode::UsePerSecond{ rate: 1.0 } )],
```

Comment:
You might want to use UseMode::Consume if you want the item to be gone right after usage.
I think the way it is currently define the item could be used 20 times a second, making the attack speed 1,000,000 :D

Reply:
Oh or alternatively, we could have cooldown + a coffee that has more than 1 "durability". This way, it would work fine with this.

Reply:
Oooh, I wanted to make a one time consumable coffee, with a 5 seconds effect duration. I'll go for the Consume instead of UsePerSecond then.